src/components/@shared/atoms/Container/index.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import Container from '@shared/atoms/Container'
+import styles from './index.module.css'
+
+export default {
+  title: 'Component/@shared/atoms/Container',
+  component: Container
+} as ComponentMeta<typeof Container>
+
+const Template: ComponentStory<typeof Container> = (args) => (
+  <Container {...args} />
+)
+
+interface Props {
+  args: {
+    narrow: boolean
```

Comment:
this is always unnecessary duplication, just import the props interface from the component. If props aren't abstracted into exportable interface in the component, then refactor them. See how the `Button` or `Alert` stories do that